watertap/property_models/multicomp_aq_sol_prop_pack.py

```diff
@@ -667,13 +667,15 @@ def initialize(
 
             # Vars indexed by phase and component_list
             for j in self[k].params.component_list:
-                if self.params.config.material_flow_basis == MaterialFlowBasis.molar:
+                if self[k].params.config.material_flow_basis == MaterialFlowBasis.molar:
```

Comment:
If this PR is taking longer time to be reviewed, can we open a tiny one to merge the correction of the two missing k's to smooth  some ongoing work?

Reply:
@lbibl @adam-a-a that sounds good to me